src/JuMP.jl

```diff
@@ -19,22 +19,16 @@ module JuMP
 
 import Base.Meta: isexpr, quot
 import LinearAlgebra
-import MathOptInterface
+import MathOptInterface as MOI
 import MutableArithmetics
 import OrderedCollections
 import OrderedCollections: OrderedDict
 import Printf
 import SparseArrays
 
+# We can't use import MutableArithmetics as _MA because of a bug in MA.
```

Comment:
```suggestion
# We can't use import MutableArithmetics as _MA because of a bug in MA.
# Fixed in MutableArithmetics v1.2.3, but would require bumping the compat
# bound so we can keep it as this until necessary.
```